openshift/template.yaml

```diff
@@ -33,7 +33,7 @@ objects:
                 containerPort: ${{WEBSERVER_PORT}}
             env:
             - name: LOG_LEVEL
-              value: ${LOG_LEVEL}
+              value: ${{LOG_LEVEL}}
```

Comment:
Env values var [must be string](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.27/#envvar-v1-core)